### ISSUE TITLE
Suggestion. MS mixin.

### ISSUE THREAD

**Issue by ghost** [adjohnson916/bootstrap-grid-ms#6]
Hello
You could add missing bootstrap add MS mixin to your less code

```
// Missing Grid mixin for Medium-Small displays
.make-ms-column(@columns; @gutter: @grid-gutter-width) {
    position: relative;
    min-height: 1px;
    padding-left:  (@gutter / 2);
    padding-right: (@gutter / 2);

    @media (min-width: @screen-ms-min) {
        float: left;
        width: percentage((@columns / @grid-columns));
    }
}
```

**Comment by AndersDJohnson** [comment]
@ddctd143 Thanks, want to submit a pull request?

**Comment by AndersDJohnson** [comment]
@ddctd143 I've implemented mixins `make-ms-column`, `make-ms-column-offset`, `make-ms-column-push`, and `make-ms-column-pull`. Please let me know if they don't work as expected.

Released as `2.0.0` to npm and Bower.
